src/services/pipewire/node.cpp

```diff
@@ -164,7 +175,7 @@ void PwNode::initProps(const spa_dict* props) {
 		}
 	}
 
-	if (this->type == PwNodeType::Audio) {
+	if (this->type & PwNodeType::Audio) {
```

Comment:
Use testFlag as its easier to verify at a glance, especially since our flag set contains combined values.